website/src/routes/index.svelte

```diff
@@ -1,28 +1,57 @@
 <script>
-  import { onMount } from "svelte";
-
-  let slideIndex = 0;
+  import {onMount, afterUpdate, onDestroy} from 'svelte';
+  import {firestore} from '../firebase';
+  let photos = [];
+
+  let slidesTimeout;
+
+  function fetchHomePhotos() {
+    return new Promise((resolve, reject) => {
```

Comment:
The  `.get()` method is already a promise, you don't need to create a promise wrapping it.